.github/ISSUE_TEMPLATE/released.yml

```diff
@@ -16,8 +26,15 @@ body:
       label: Version
       description: What version are you running that you noticed this problem?
       options:
-        - 4.3.0 (Current)
+        - 4.5.0 (Current)
+        - 4.4.2
+        - 4.4.1
+        - 4.4.0
+        - 4.3.0 
         - 4.2.0
+        - 4.1.0
+        - 4.0.3
+        - 4.0.0
         - 3.11.0
```

Comment:
The version should refer to the V2Xhub version I think. Maybe from 7.8.1 to 7.0?